articles/proofs-arguments-and-zero-knowledge/facts/Schwartz-Zippel-Lemma.md

```diff
@@ -0,0 +1,4 @@
+### Schwartz-Zippel Lemma
```

Comment:
Should add something like:
The total degree of  m-variate polynomial is the number of possible root

Reply:
I do not clearly understand what I should add.

Reply:
Let $F$ be any field, and let g: $F^m → F$ be a nonzero $m$-variate polynomial of total degree (a.k.a the number of possible root) at most $d$. Then on any finite set $S \subseteq F$ : $$Pr_{x←S^m} [g(x) = 0] ≤ d/|S|.$$